Algorithm.Framework/Portfolio/MeanVarianceOptimizationPortfolioConstructionModel.cs

```diff
@@ -246,11 +246,8 @@ protected override bool ShouldCreateTargetForInsight(Insight insight)
             // Get symbols' returns
```

Comment:
Let's improve this a comment on using linear returns according to

> Meucci, Attilio, Quant Nugget 2: Linear vs. Compounded Returns – Common Pitfalls in Portfolio Management (May 1, 2010). GARP Risk Professional, pp. 49-51, April 2010 , Available at SSRN: https://ssrn.com/abstract=1586656

Same for Python.